apps/opik-frontend/src/constants/shared.ts

```diff
@@ -32,3 +33,5 @@ export const CELL_HORIZONTAL_ALIGNMENT_MAP: Record<COLUMN_TYPE, string> = {
   [COLUMN_TYPE.dictionary]: "justify-start",
   [COLUMN_TYPE.numberDictionary]: "justify-start",
 };
+
+export const OPIK_URL_OVERRIDE_CONFIG = `os.environ["OPIK_URL_OVERRIDE"] = "${window.location.origin}${BASE_API_URL}"`;
```

Comment:
don't you think we should take some of the env variables instead of `{window.location.origin}`?

what do you think, @ferc ?

Reply:
or i would merge `window.location.origin` and `BASE_API_URL` more sophisticatedly let's say. with `new URL()` for example

Reply:
just to handle the cases where `BASE_API_URL` is absolute and relative - starts with `https://` or `/...`